src/main/java/com/ge/predix/uaa/token/lib/FastTokenServices.java

```diff
@@ -376,4 +393,8 @@ public void setMaxAcceptableClockSkewSeconds(final int maxAcceptableClockSkewSec
     public void setTrustedIssuers(final List<String> trustedIssuers) {
         this.trustedIssuers = trustedIssuers;
     }
-}
+
+    public void setTimeToLiveMillis(final long timeToLiveMillis) {
```

Comment:
this ttl value will never be used. if there is no need to update the TTL for a existing bean, pls remove the setter.

Reply:
i forgot this one ... I had it at the back of my mind to remove it.